gse-util/src/main/java/com/powsybl/gse/util/NodeChooser.java

```diff
@@ -620,14 +649,94 @@ private void renameNode(TreeItem selectedTreeItem) {
         });
     }
 
+    private MenuItem createCopyProjectNodeItem(List<? extends TreeItem<N>> selectedTreeItems) {
+        MenuItem copyMenuItem = GseMenuItem.createCopyMenuItem();
+        copyMenuItem.setDisable(selectionContainsNonArchiveFolder(selectedTreeItems));
+        copyMenuItem.setOnAction(event -> copy(selectedTreeItems));
+        return copyMenuItem;
+    }
+
+    private boolean selectionContainsNonArchiveFolder(List<? extends TreeItem<N>> selectedTreeItems) {
+        return selectedTreeItems.stream().map(treeItem -> (N) treeItem.getValue())
+                .filter(val -> val instanceof Folder && !((Folder) val).isWritable())
+                .map(folder -> new java.io.File(((Folder) folder).getPath().toString().replace("/:", "")))
+                .anyMatch(this::isNotAnArchiveFolder);
+    }
+
+    private boolean isNotAnArchiveFolder(java.io.File file) {
+        java.io.File[] files = file.listFiles();
+        return files != null && Arrays.stream(files).noneMatch(file1 -> file1.getName().equals("info.json"));
+    }
+
+    private MenuItem createPasteProjectNodeItem(TreeItem<N> selectedTreeItem) {
+        MenuItem pasteMenuItem = GseMenuItem.createPasteMenuItem();
+        pasteMenuItem.disableProperty().bind(copied.not());
+        final Clipboard clipboard = Clipboard.getSystemClipboard();
+        pasteMenuItem.setOnAction(event -> {
+            if (selectedTreeItem.getValue() instanceof Folder) {
+                Folder destinationFolder = (Folder) selectedTreeItem.getValue();
+
+                if (clipboard.hasString()) {
```

Comment:
Same remark as the one made in `ProjectPane`, the handling of the clipboard content for copy should be a static method in the business class `CopyManager`.